### DISCUSSION TITLE
Fission protection

### DISCUSSION THREAD

**Discussion by rsssxd** [mozilla-mobile/fenix#24485] [Q&A] [answered]
Desktop Firefox recently got Fission protection.

When will Firefox for Android have Fission protection?

Any progress on this?

**Comment by kbrosnan** [comment, marked as answer]
Ask the Fission team. No one on this project works on the Gecko platform. https://chat.mozilla.org/#/room/#fission:mozilla.org

**Comment by rsssxd** [comment]
Fission protection is running on the latest night builds.

![Screenshot_20220401-145939_Firefox_Nightly](https://user-images.githubusercontent.com/60405617/162290157-50cfbd4b-6a0a-4253-9273-4bed07a7ca04.png)
